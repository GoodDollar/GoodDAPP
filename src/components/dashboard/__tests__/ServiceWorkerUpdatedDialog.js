import React from 'react'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

import { getWebRouterComponentWithMocks } from './__util__'

describe('ServiceWorkerUpdatedDialog', () => {
  it('renders without errors', () => {
    const ServiceWorkerUpdatedDialog = getWebRouterComponentWithMocks('../ServiceWorkerUpdatedDialog')
    const tree = renderer.create(<ServiceWorkerUpdatedDialog />)
    expect(tree.toJSON()).toBeTruthy()
  })

  it('matches snapshot', () => {
    const ServiceWorkerUpdatedDialog = getWebRouterComponentWithMocks('../ServiceWorkerUpdatedDialog')
    const component = renderer.create(<ServiceWorkerUpdatedDialog />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
