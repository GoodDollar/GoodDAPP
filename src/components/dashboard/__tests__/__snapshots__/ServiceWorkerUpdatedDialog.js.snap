// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceWorkerUpdatedDialog matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n"
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": " center",
          "WebkitBoxAlign": " center",
          "alignItems": " center",
          "display": "flex",
          "msFlexAlign": " center",
          "paddingLeft": "5px",
          "paddingRight": "5px",
          "paddingTop": "32px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
        style={
          Object {
            "height": "134px",
            "marginBottom": "25px",
            "textAlign": "center",
            "width": "191px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
          style={
            Object {
              "backgroundImage": "url(\\"updateversion.svg\\")",
            }
          }
        />
        <img
          alt=""
          className="css-accessibilityImage-9pa8cd"
          draggable={false}
          src="updateversion.svg"
        />
      </div>
      <div
        className="css-text-901oao r-width-13qz1uu"
        dir="auto"
        style={
          Object {
            "borderBottomColor": "rgba(0,175,255,1.00)",
            "borderBottomStyle": "solid",
            "borderBottomWidth": "2px",
            "color": "rgba(66,69,74,1.00)",
            "fontFamily": "Roboto",
            "fontSize": "22px",
            "fontWeight": "700",
            "lineHeight": "26px",
            "marginBottom": "9px",
            "paddingBottom": "9px",
            "textAlign": "left",
            "textDecoration": "none",
            "textTransform": "none",
          }
        }
      >
        Hip hip hooray!
We have a new version :)
      </div>
    </div>
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "minHeight": "100px",
        }
      }
    >
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "color": "rgba(66,69,74,1.00)",
            "fontFamily": "Roboto",
            "fontSize": "14px",
            "fontWeight": "normal",
            "lineHeight": "20px",
            "textAlign": "left",
            "textDecoration": "none",
            "textTransform": "none",
          }
        }
      >
        Thanks to your awesome feedback we were able to improve our wallet. We hope now youâ€™ll enjoy it even more.
      </div>
    </div>
  </div>
</div>
`;
